pkg/util/ec2/ec2.go

```diff
@@ -169,31 +177,45 @@ func doHTTPRequest(url string, method string, headers map[string]string, retriab
 	if err != nil {
 		return nil, err
 	}
+
+	if useToken {
+		token, err := getToken()
+		if err != nil {
+			log.Info("ec2_prefer_imdsv2 is set to true in configuration but the agent was unable to get a token: %s", err)
```

Comment:
Shouldn't this message be a warning or an error?

Reply:
That's a good point, as this code can only be reached if the default configuration has been changed, I think that logging this as a warning is reasonnable.